CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Version *next* (not yet released)
 
+- Reorganize repo structure.
+- Add tests and CI support.
```

Comment:
This should definitely go in the changelog!

Reply:
Agreed! Done.